.vscode/settings.json

```diff
@@ -14,5 +14,6 @@
   "tslint.exclude": "**/node_modules/**",
   "tslint.autoFixOnSave": true,
   "tslint.jsEnable": true,
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "postcss.validate": false
```

Comment:
As we don't advocate the use of any particular plugins beyond tslint/typescript, maybe remove this?